core/templates/services/keyboard-shortcut.service.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Keyboard shortcut service for Oppia webpages.
+ */
+import 'mousetrap';
+
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class KeyboardShortcutService {
+  bindExplorationPlayerShortcuts() {
+    Mousetrap.bind('s', function() {
+      document.getElementById('skipToMainContentId').focus();
+      return false;
+    });
+
+    Mousetrap.bind('k', function() {
+      var previousButton = document.getElementById('backButtonId');
```

Comment:
Use DOCUMENT maybe? (a wrapper for the document in angular)

Reply:
Is it important to use wrapper? It seems in other files they just use the base document. see core/templates/services/search.service.spec.ts.